docs/index.md

```diff
@@ -37,7 +37,28 @@ Furthermore, during data processing and training, the data is stored on UCloud w
 
 ## Improving the Danish Language Technology Landscape
 
-The Danish Foundations models collaborate with the [Danish Data Science Community](https://danskdatascience.dk/), [Centre for Humanities Computing Aarhus](https://chcaa.io/), [The Alexandra Institute](https://alexandra.dk) to promote the development of Danish language tools. We continually gather information about how to improve the Danish language technologies and how to best support the community. To this end we have created a [list of missing pieces for Danish NLP](https://github.com/centre-for-humanities-computing/danish-foundation-models/discussions/categories/missing-pieces-for-danish-nlp) and we invite any 1) to add to the list, 2) solve one of the problems or 3) upvote the problems you think are most important.
+The Danish Foundations models collaborate with the [Danish Data Science Community](https://danskdatascience.dk/), [Centre for Humanities Computing Aarhus](https://chcaa.io/), [The Alexandra Institute](https://alexandra.dk), and [Center for AI Science and Applications](https://sdu.dk/casa) to promote the development of Danish language tools. We continually gather information about how to improve the Danish language technologies and how to best support the community. To this end we have created a [list of missing pieces for Danish NLP](https://github.com/centre-for-humanities-computing/danish-foundation-models/discussions/categories/missing-pieces-for-danish-nlp) and we invite any 1) to add to the list, 2) solve one of the problems or 3) upvote the problems you think are most important.
+
+
+## The Team
+
+- From [the Center for Humanities Computing at Aarhus University](https://chc.au.dk/):
+  - Kenneth Enevoldsen (kenneth.enevoldsen@cas.au.dk)
+  - Lasse Hansen (lasse.hansen@clin.au.dk)
+  - Martin Bernstorff (martinbernstorff@gmail.com)
```

Comment:
Want to use your clin email as well @MartinBernstorff or prefer gmail?

Reply:
I used the mail on notion, but @MartinBernstorff feel free to change it